opensfm/exif.py

```diff
@@ -176,10 +181,25 @@ def extract_focal(self):
         focal_35, focal_ratio = compute_focal(
             get_tag_as_float(self.tags, 'EXIF FocalLengthIn35mmFilm'),
             get_tag_as_float(self.tags, 'EXIF FocalLength'),
-            get_tag_as_float(self.tags, 'EXIF CCD width'),
+            self.extract_sensor_width(),
```

Comment:
By doing this, we are no longer reading the `EXIF CCD width` tag.  Should `extract_sensor_width()` handle the `EXIF CCD width` tag?

Reply:
Awesome, wonder how we got look for it. Was just checking now and could not find any reference of it anywhere. Thanks for looking at this.